presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -235,6 +235,26 @@ Network Address
 
     Examples: ``IPADDRESS '10.0.0.1'``, ``IPADDRESS '2001:db8::1'``
 
+.. _ipprefix_type:
+
+``IPPREFIX``
+^^^^^^^^^^^^
+
+    An IP routing prefix that can represent either an IPv4 or IPv6 address.
```

Comment:
I think this format is better than the `IPADDRESS` one. Maybe you can add another commit to reformat `IPADDRESS` to be consistent.